README
To install this ppm package, run the following command
in the current directory:

   ppm install SOAP-Lite-0.70_04.ppd



Changes
#
# Revision history for Perl extension SOAP::Lite.
#

The contents of this file provides a change history for the SOAP::Lite
Perl module. New features are designated with a '+' and bug fixes with
a '!'.
Numbers in square brackets are sourceforge tracker IDs from
https://sourceforge.net/tracker/?group_id=66000&atid=513017

Numbers with # are CPANRT tracker IDs from
http://rt.cpan.org/Public/Dist/Display.html?Name=SOAP-Lite

-----------------------------------------------------------------------
THIS RELEASE
-----------------------------------------------------------------------
1.28 August 22th
    ! Extracted CGI transport conditional read into subclass

1.27 May 14, 2018
    ! Problem with compression in server mode

1.26 December 30, 2017
    ! typo fix https://anonscm.debian.org/cgit/pkg-perl/packages/libsoap-lite-perl.git/tree/debian/patches/spelling-error.patch

1.25 December 29, 2017
    ! Bad usage of UNIVERSAL::isa (See RT#62667)

1.24 December 19, 2017
    + Consistent Versioning https://github.com/redhotpenguin/soaplite/pull/24
    ! Remove is_utf8() check from 78588

1.23 December 18, 2017
    ! RT #78588 unicode data not correct encoded

1.22 August 15, 2017
    ! Noindex URI::tcp module [github/JaSei]
    ! Correct the soap namespace declaration when calling services defined in WSDL [github/waaeer]
    ! Fix typos [github/kmhl]

1.21 November 6, 2016
    ! #118654 consistently use '#!/usr/bin/env perl' throughout (@mototimo) [github/moto-timo]

1.20 June 9, 2016
    ! Minor kwalitee updates (@oeuftete)
    ! #106688 encode utf8 content in test which fails under RHES 6.6

1.19 August 26, 2015
    ! #106665 Correct the version manually, no autoversioning, and the maintainer screwed up

1.18 August 25, 2015
    ! Fix a couple of cases where undefined values were being passed to functions [github/redhotpenguin]
    ! Do not set content_length for compressed request [github/bor]
    ! Add test for check return of send_receive(), add titles for some tests [github/bor]

1.17 July 30, 2015
    ! #106051 Fix utf8 encoding for CGI test (thanks cpan/ETHER for tips)

1.16 July 23, 2015
    ! Fix empty release. Note to self - do not release code when wife is in early labor

1.15 July 21, 2015
    + Preventing "billion laughs" exponential entity expansion attack [github/thilo-fester]

1.14 March 24, 2015
    + Add header and body tags [@bubi6608]
    ! Fixed typo in trace logging example [@oschwald]
    + Converted the dist from using ExtUtils::MakeMaker to using Dist::Zilla [@mvuets]
    ! Cleaned up prereqs [@mvuets]
    ! Cleaned up author and release tests [@mvuets]

1.13 December 30, 2014
    ! Fixed undefined value error when using bugzilla's xmlrpc.cgi (bugzilla's API) [dbeusee]

1.12 November 26, 2014
    ! #100557 Redefined sub in SOAP::Lite::Deserializer::XMLSchema2001

1.11 February 21, 2014
    ! #93080 bug when SOAP::Lite is trying to parse WSDL definition by https with custom settings

1.10 January 23, 2014 
    ! Fixed uninitialized warning when Context-Length is not set [github/dbeusee]
    ! avoid string eval in as_base64 [github/otrosien]
    ! #88915 warnings issued with using importing with +trace

1.09 January 14, 2013
    ! #92025 tighten multipart content-type matching [dmn@debian.org]

1.08 November 8, 2013
    ! #15703 add no_index for examples to prevent CPAN pollution [githumb.com/sergeyromanov]

1.07 November 7, 2013
    ! #89948 Remove warning when Content-Id is not present

1.06 August 21, 2013
    ! #87935 skip all MIME-Parser tests if it is not installed
    ! #87787 Use of uninitialized value $ns in Lite.pm 

1.05 August 18, 2013
    ! #87848 Fix manifest, fix invalid http version from release blunder

1.04 August 9, 2013
    ! #87605 Skip XML::Parser::Lite tests if it is missing [github.com/omega] 

1.03 August 4, 2013
    ! #87195 Remove DIME::Tools dep until we can work around its issues

1.02 July 30, 2013
    ! #87421 1.01 release blew out xml versions. mea culpa.

1.01 July 29, 2013
    ! #84168 Test t/02-payload.t fails under Perl 5.17.10
    ! #87127 TEST.pl removed from MANIFEST
    ! revert #85713 - various reports of environment failures

1.0 July 16, 2013
    ! #85713 SOAP::Transport::HTTP, 500 error skips parsing of response
    ! No more Windows 98 client support. Wait, wut?
    ! Merge SOAP::Transport::TCP back into SOAP::Lite (for now)
    ! #82416: xop href cid: URI encoded support reported by Michael R. Davis (mrdvt92)
    ! #85208 bad test plan
    ! #83750 different headers for mod_perl vs mod_perl2 [Mithun Bhattacharya]

0.716 May 10, 2013
    ! #17275 Client unable to detect fatal SOAP failure (e.g. unable to connect to host)
    ! #81471 Force array context for correct Apache header parsing in SOAP::Transport::HTTP
    ! #45611 deserializing SOAP message with composite attachment raises exception
    ! #84168 Test t/02-payload.t fails under Perl 5.17.10
    ! #85098 Monkey patch for LWP::Protocol confuses the toolchain
    ! #78692 / #84724 / #83715 Sending large object (many levels deep) throws error Incorrect parameter
    ! #78502 t/08-schema.t noisy under 5.17.2 (unescaped left brace in regex  is deprecated)
    ! #78608 Documentation for "+trace"
    ! #78639 Spelling error reported by Debian tools
0.715
    ! #75938 Change to warning / add configuration variable to allow XML tags starting with /xml/i
    ! #76902 Missing info in "SSL CERTIFICATE AUTHENTICATION" documentation
	! #74577 work around/fixes for success with MS WS https/compressed
    ! #74308 SOAP-Lite fails for operations with period in name
    ! #73802 client did not set the encoding of the soap-envelope to 'iso-8859-1' in HTTP-request
    ! #75019 HTTP server doesn't process PIPE signal
    ! #72836 Processing a SOAP 1.2 request breaks the next SOAP 1.1 response
    ! #70381 SOAP-Lite performance
    ! #53375 comments after <?xml .. >
    ! #63693 Uninitalized value (again)
    ! #37360 Use of uninitialized variables in SOAP tests

Makefile.PL
# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v5.034.
use strict;
use warnings;



use ExtUtils::MakeMaker;

my %WriteMakefileArgs = (
  "ABSTRACT" => "Perl's Web Services Toolkit",
  "AUTHOR" => "Pavel Kulchenko, Randy J Ray, Byrne Reese, Martin Kutter <MKUTTER\@cpan.org>, Fred Moyer",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0
  },
  "DISTNAME" => "SOAP-Lite",
  "EXE_FILES" => [
    "bin/SOAPsh.pl",
    "bin/stubmaker.pl"
  ],
  "LICENSE" => "perl",
  "NAME" => "SOAP::Lite",
  "PREREQ_PM" => {
    "Class::Inspector" => 0,
    "Compress::Zlib" => 0,
    "IO::SessionData" => "1.03",
    "IO::Socket::SSL" => 0,
    "LWP::Protocol::https" => 0,
    "LWP::UserAgent" => 0,
    "MIME::Base64" => 0,
    "Scalar::Util" => 0,
    "Task::Weaken" => 0,
    "URI" => 0,
    "XML::Parser" => "2.23",
    "constant" => 0
  },
  "TEST_REQUIRES" => {
    "IO::File" => 0,
    "Test::More" => 0,
    "Test::Warn" => 0,
    "XML::Parser::Lite" => 0
  },
  "VERSION" => "1.15",
  "test" => {
    "TESTS" => "t/*.t t/Issues/*.t t/SOAP/*.t t/SOAP/Lite/*.t t/SOAP/Lite/Deserializer/*.t t/SOAP/Schema/*.t t/SOAP/Transport/*.t t/SOAP/Transport/HTTP/*.t"
  }
);


my %FallbackPrereqs = (
  "Class::Inspector" => 0,
  "Compress::Zlib" => 0,
  "ExtUtils::MakeMaker" => 0,
  "IO::File" => 0,
  "IO::SessionData" => "1.03",
  "IO::Socket::SSL" => 0,
  "LWP::Protocol::https" => 0,
  "LWP::UserAgent" => 0,
  "MIME::Base64" => 0,
  "Scalar::Util" => 0,
  "Task::Weaken" => 0,
  "Test::More" => 0,
  "Test::Warn" => 0,
  "URI" => 0,
  "XML::Parser" => "2.23",
  "XML::Parser::Lite" => 0,
  "constant" => 0
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);
